### ISSUE TITLE
jupyter-notebook of the mask reconstruction

### ISSUE THREAD

**Issue by JunMa11** [kevinjohncutler/omnipose#21]
Thanks again for the awesome work.

Could you please share a jupyter-notebook to show 
1. how to generate the two flow components, the distance field and a boundary field based on ground truth?
2. how to reconstruct the mask based on the above elements?

**Comment by kevinjohncutler** [comment]
@JunMa11 good idea, that would be instructive. I have some very unpolished versions of this already that I can clean up, but it will have to wait for late next week. If you want to dig into it sooner, `omnipose.core` contains these functions. Specifically, `labels_to_flows()` and `compute_masks()`.
